plugins/inventory/aws_ec2.py

```diff
@@ -64,6 +64,13 @@
         type: str
         default: '_'
         required: False
+  allow_duplicated_hosts:
+    description:
+      - By default, only the first name that back the I(hostnames) list is returned.
+      - Turn this flag on if you don't mind having duplicated entries in the inventory
+        and you want to get all the hostnames that match.
+    type: bool
+    default: False
```

Comment:
Please add `version_added: 5.0.0`

Reply:
does anything speak against releasing it with a 4er version?